riff-raff/app/persistence/RestrictionConfigDynamoRepository.scala

```diff
@@ -0,0 +1,22 @@
+package persistence
+
+import java.util.UUID
+
+import com.gu.scanamo.Table
+import restrictions.{RestrictionConfig, RestrictionsConfigRepository}
+
+object RestrictionConfigDynamoRepository extends DynamoRepository with RestrictionsConfigRepository {
```

Comment:
Testing is non-trivial without it. Having a local dynamodb for tests would also eliminate it.